WordPress/src/main/java/org/wordpress/android/viewmodel/mlp/ModalLayoutPickerViewModel.kt

```diff
@@ -161,9 +168,14 @@ class ModalLayoutPickerViewModel @Inject constructor(
     /**
      * Shows the MLP
      * @param supportedBlocks the supported blocks to filter fetched layouts (by default no filtering occurs)
+     * @param previewWidth the layout preview card width
+     * @param scale the screen density scale
```

Comment:
`scale` is mainly used for iOS. We use the default value 1.0 and pass the rendered pixels per device in `previewWidth`